packages/perseus/src/widgets/numeric-input/utils.ts

```diff
@@ -0,0 +1,121 @@
+import type {PerseusStrings} from "../../strings";
+import type {PerseusNumericInputAnswerForm} from "@khanacademy/perseus-core";
+
+/**
+ * The full list of available strings for the numeric input widget,
+ * based on whether the Content Creator has specified that the answer must be simplified.
+ */
+export const NumericExampleStrings: {
+    [key: string]: (
+        form: PerseusNumericInputAnswerForm,
+        strings: PerseusStrings,
+    ) => string;
+} = {
+    integer: (form, strings: PerseusStrings) => strings.integerExample,
```

Comment:
If these are typed on line 11, why do we have to type `strings` for each function?

Reply:
I'm fairly certain that I didn't actually change this, it was just copied over from the original implementation. There's likely no reason for how specific this is.